<?xml version="1.0"?>
<launch>
    <!-- AprilTag Detection Node -->
    <node name="apriltag_detector" pkg="myapriltags" type="apriltag_detector.py" output="screen">
        <param name="stop_distance_threshold" value="0.3" />
        <param name="tag_size" value="0.065" />
    </node>
    
    <!-- AprilTag Controller Node -->
    <node name="apriltag_controller" pkg="myapriltags" type="apriltag_controller.py" output="screen">
        <param name="stop_duration" value="2.0" />
    </node>
</launch>